conversiontracerbullet/src/main/java/org/stanford/AuthDBProperties.java

```diff
@@ -19,22 +16,22 @@
 
     private static Logger log = LogManager.getLogger(AuthDBProperties.class.getName());
 
-    private static final String DEFAULT_CONFIG_FILE = "/server.conf";
+    private static final String PROPERTY_RESOURCE = "/server.conf";
 
     private String server = null;
     private String service = null;
     private String userName = null;
     private String userPass = null;
-    private String propertyFile = null;
+    private Properties properties = null;
```

Comment:
would it make more sense to pass "properties" as an argument to initDataSourceProperties(), since we call it in the constructor(s) and we must instantiate properties before calling it?